src/api.cc

```diff
@@ -1061,7 +1062,7 @@ std::unique_ptr<InstallContext> AkliteClient::Installer(const TufTarget& t, std:
   }
   if (target == nullptr) {
     const auto uptane_target{Target::fromTufTarget(t)};
-    if (Target::isInitial(uptane_target) && client_->wasTargetInstalled(uptane_target)) {
+    if ((!require_target_in_tuf || Target::isInitial(uptane_target)) && client_->wasTargetInstalled(uptane_target)) {
```

Comment:
I would update the comment below with info about this use-case too.

Reply:
@mike-sul  comment updated. Can you take a look to see it if is OK?
Is there any other situation, other than a rollback, that the "Initial target" could be used? If not, perhaps we can remove the `Target::isInitial(uptane_target)` test, and leave only `require_target_in_tuf`

Reply:
>Is there any other situation, other than a rollback, that the "Initial target" could be used? If not, perhaps we can remove the Target::isInitial(uptane_target) test, and leave only require_target_in_tuf

Yes, there are such situations, e.g. customer can play with sql.db, or remove the `installed_versions` file, so then we can end up doing a regular update or sync for "initial target" which might not be among targets listed in the local tuf db.